module3/preparation/Week1/CSSFundamentals.md

```diff
@@ -0,0 +1,112 @@
+---
+layout: page
+title: "Intro to CSS: Pre-Work"
+---
+
+## What is CSS?
+
+CSS a “style sheet language” that lets you style the HTML elements on your page. CSS works _with_ HTML, but isn't HTML. CSS controls the positioning, sizing, colors, and specific fonts on your page.
+
+## Three Ways to Implement CSS
+
+There are multiple ways to implement CSS:
+1. Inline CSS
+2. Internal CSS
+3. External CSS
+
+Modern applications utilize **External CSS**, as we will during your time at Turing. An External StyleSheet is when we have a CSS file (ends with the file extension `.css`) that is linked to the HTML document. It's good to have a little background on the other two methods as you may occasionally come across examples in documentation or Stack Overflow posts. You can read up on them [here](https://www.freecodecamp.org/news/get-started-with-css-in-5-minutes-e0804813fc3e/).
+
+## Syntax & Style
+
+There can be broad variations in how an external CSS file looks, but still do the same thing. As developers, half of the challenge of our job is to **make sure code is readable and maintainable** - by others and future you. This is why clean and consistent syntax and style is essential to your practice. Building the habit of taking the time to write clean code is a huge part of your work in Module 1.
+
+## Commonly Used Properties
+
+Go to either [MDN Web Docs](https://developer.mozilla.org/en-US/) or [CSS Tricks](https://css-tricks.com/) and get familiar with the following CSS properties, and some of the possible values that could be provided each one.
+
+- background-color
+- border
+- border-radius
+- box-shadow
+- color
+- cursor
+- font-family
+- font-size
+- height
+- opacity
+- width
+
+## Selectors: Classes and IDs
+
+HTML elements can be referenced in the External CSS file in multiple ways. The most straightforward method is by referencing the element itself:
+
+```html
+<h1>Hello, World!</h1>
+```
+
+```css
+h1 {
+  color: magenta;
+}
+```
+
+However, this can create problems when a page has multiple elements from the same tag, but we want to style those elements differently.
+
+There is a `class` and `id` attribute available to use on __every__ HTML element. These attributes allow you to create "targets" that are more specific than elements. They are hooks that you can use to manipulate the look and behavior of your HTML elements.
+
+### Classes
```

Comment:
Actually now that I've looked at the lesson, might be helpful to introduce classes and Ids as part of the prep. Not thinking about using them as selectors, but just how classes and Ids work.